### DISCUSSION TITLE
6 Project Ideas

### DISCUSSION THREAD

**Discussion by gabebott** [USU-ITLS-x265-23-summer/class-repo#66] [Week 2 - Discussions]
1. Personal Portfolio Website: This website would help me to display my UX portfolio and also showcase my abilities to program and code.
2. Social Media Dashboard: I have seen this idea before and I think it would be really cool to create a site that combines all of the feed from your different social media accounts.
3. Ecommerce Store: I would like to build a site that allows users to search and buy products from my store.
4. Sports Blog App: An app where people can talk about their favorite sports teams in dedicated sports forums
5. To-do list application: A simple app that works as a to do list to check off accomplishments and tasks
6. Game development: I would like to create a simple game of some sort.

**Comment by Skye404** [comment]
I like your ideas. They're creative. Some of them seem like they would take a lot of work probably beyond what this class covers, but they are all super interesting. I haven't heard of something like your #2, but that would be really interesting to see something like that.

**Comment by metasean** [comment]
- Personal Portfolio Website || Sports Blog App
  - These are relatively safe, utilitarian projects.
  - You can make each of them as easy or challenging as you want.
  - You can also make each of them as vanilla or creative as you want
    - e.g. https://enhancv.com/blog/the-23-most-creative-resume-designs-weve-seen/

- Social Media Dashboard
  - This could definitely be a fun project.
  - It would require going a bit beyond the material we'll cover in this class.  If you were in 6265, this would be a perfect stretch, but it's a bit more than I expect to be demonstrated at the 4265 level.

- Ecommerce Store
  - I'd definitely avoid this, as it is very likely, to get very complex, very quickly.

- To-do list application
  - This is one of the most common developer toy apps in the world. 😁  

- Game development
  - Simple online games, can be, well simple.
    - https://devdevshow.com/javascript-games-examples-for-beginners/
    - https://webtips.dev/javascript-game-ideas
